### PR TITLE
refactor : Input

### PR DESCRIPTION
**Pull request by dikshadhote**
Refracted code for 
1.input field
2.label
3.code snippet

image
image

### CODE REVIEW

File: components/input.css [hunk 1]
```diff
@@ -6,10 +6,14 @@
   border-radius: 4px;
 }
 .input {
-  padding: 5px;
-  margin: 1rem;
-  text-align: center;
+  padding: 5px 10px 5px 10px;
+  margin: 8px 1rem 8px 1rem;
```
Comment (Namrata-J) [inline components/input.css]: you can simply apply padding: 5px 10px; and margin: 8px 1rem; and it will work the same.

**Review by Namrata-J — APPROVED** [review]

File: documents/documents.html [hunk 2]
```diff
@@ -775,36 +775,30 @@ <h2>Inputs</h2>
...
-             
-              sandbox="allow-scripts allow-same-origin"
-            >
-            </iframe>
+            src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522input-container%2520mt-2%2520ml-1%2520d-flex%2520flex-column%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522input-label%2522%2520class%253D%2522ml-2%2520input-label%2522%253EEmail%253C%252Flabel%253E%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520class%253D%2522input%2522%2520id%253D%2522input-label%2522%2520placeholder%253D%2522you%2540example.com%2522%252F%253E%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520"
```
Comment (Kothalkarpriya) [inline documents/documents.html:785]: inline styles should be avoided

File: documents/documents.html [hunk 3]
```diff
@@ -775,36 +775,30 @@ <h2>Inputs</h2>
...
+  src="https://carbon.now.sh/embed?bg=rgba%2821%2C23%2C24%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=100px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522input-container%2520mt-2%2520ml-1%2520d-flex%2520flex-column%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520class%253D%2522ml-2%2520input-label%2522%2520for%253D%2522inp-password%2522%253EPassword%253C%252Flabel%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522inp-password%2522%2520class%253D%2522input%2520input-error%2522%2520%2520placeholder%253D%2522*******%2522%2520%252F%253E%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Csmall%2520class%253D%2522text-red%2520ml-2%2522%253EForgot%2520your%2520password%253F%253C%252Fsmall%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520"
+  style="width: 813px; height: 169px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Kothalkarpriya) [inline documents/documents.html]: here also

**Review by Kothalkarpriya — APPROVED** [review]
Look into My comments, then you can merge

File: documents/documents.html (hunk 2 above)
Comment (dikshadhote) [reply documents/documents.html:785]: Removed the inline style

File: documents/documents.html (hunk 3 above)
Comment (dikshadhote) [reply documents/documents.html]: Removed!

File: components/input.css (hunk 1 above)
Comment (dikshadhote) [reply components/input.css]: Yeap ! changed to shorthand

**Pull request merged**